src/sentry/tasks/autofix.py

```diff
@@ -32,3 +33,19 @@ def check_autofix_status(run_id: int):
         logger.error(
             "Autofix run has been processing for more than 5 minutes", extra={"run_id": run_id}
         )
+
+
+@instrumented_task(
+    name="sentry.tasks.autofix.start_seer_automation",
+    max_retries=1,
+    taskworker_config=TaskworkerConfig(
+        namespace=issues_tasks,
+        retry=Retry(
+            times=1,
+        ),
+    ),
+)
+def start_seer_automation(group: Group):
```

Comment:
You should pass a `group_id` here rather than a `Group`, and then fetch from postgres on this side. cc @markstory, do we do any automated translation for this in the new system?

Reply:
> do we do any automated translation for this in the new system?

No, tasks need to be manually updated to include `taskworker_config` metadata.

Reply:
> You should pass a group_id here rather than a Group, and then fetch from postgres on this side.

I made this change now.